packages/blade/src/components/Card/CardHeader.tsx

```diff
@@ -258,6 +276,16 @@ const CardHeaderTrailing = assignWithoutSideEffects(_CardHeaderTrailing, {
   componentId: ComponentIds.CardHeaderTrailing,
 });
 
+const _CardHeaderBox = (props: BoxProps): React.ReactElement => {
+  useVerifyInsideCard('CardHeaderBox');
+
+  return <BaseBox {...props} />;
+};
+
+const CardHeaderBox = assignWithoutSideEffects(_CardHeaderBox, {
+  componentId: ComponentIds.CardHeaderBox,
+});
```

Comment:
Should we drop this and just allow Box component itself? Since box is a very common component that people might intuitively use instead of this and this is also a pattern we can eventually get rid of and make card more flexible